wolfcrypt/src/memory.c

```diff
@@ -521,7 +521,7 @@ struct wc_Memory {
 /* returns amount of memory used on success. On error returns negative value
    wc_Memory** list is the list that new buckets are prepended to
  */
-static int create_memory_buckets(byte* buffer, word32 bufSz,
+static int wc_create_memory_buckets(byte* buffer, word32 bufSz,
```

Comment:
If these are still static, is there a reason to rename?  Or are you expecting to make the non-static in the future?

Reply:
I renamed this one because most of the other static functions in the file still had `wc` prefixes. There's no plan to make the non-static. I was just adding a little namespace to it.